tests/test_write.py

```diff
@@ -77,8 +77,14 @@ def test_write_zarr_2d(self):
             # Use the temporary directory
             test_file_path = os.path.join(dir, 'out/test.ome.zarr')
 
-            bw = TSWriter(test_file_path, tmp.shape, tmp.shape, str(tmp.dtype))
-            bw.write_image_data(tmp)
+            rows = Seq(0, br._Y - 1, 1)
+            cols = Seq(0, br._X - 1, 1)
+            layers = Seq(0, 0, 1)
+            channels = Seq(0, 0, 1)
+            tsteps = Seq(0, 0, 1)
```

Comment:
Add a test where you are writing two different part of the image. May be a 2D image where you write two part of the images separately.
Also, another 3D image, where you write the channels separately.